### PR TITLE
Newsletter-252: translate into Chinese

### PR DESCRIPTION
**Pull request by hulatown**

### CODE REVIEW

File: _includes/specials/policy/zh/02-cache-utility.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+交易池验证在处理花费相同输入的交易（即，重复花费或者说冲突交易）时也会用到手续费和手续费率。节点不会按先到先得的规则保留先收到的那一笔交易，而是会根据一组规则来确定哪一笔交易更加激励兼容，然后保留下来。这个行为叫做 “手续费替换（[Replace by Fee][topic rbf]）”。
+
+矿工会最大化手续费，这符合直觉，但为什么一个不挖矿的节点也要实现这些规则呢？就像上周本栏目所说的，不挖矿节点的交易池的效用，是由它跟矿工的交易池的相似性决定的。因此，即使一个节点从不尝试使用其交易池中的内容产生区块，他们也愿意保存最为激励兼容的交易。
+
+虽然没有共识规则要求交易支付手续费，但手续费和手续费率在为比特币网络中扮演着重要角色，它是一种 “公平的” 解决区块空间竞争的办法。矿工使用手续费率来评估可取程度、处理驱逐和冲突，而不挖矿的节点也参照这种行为，以尽可能提高自己的交易池的效用。
+
+区块空间的稀缺性，产生了降低交易体积的压力，并鼓励开发者构造更有效率的交易。下周的专栏中，我们会探索链上交易节约手续费的实用策略和技术。
+
+[policy01]: /en/newsletters/2023/05/17/#waiting-for-confirmation-1-why-do-we-have-a-mempool
```
Comment (bitschmidty) [inline _includes/specials/policy/zh/02-cache-utility.md:23]:
```suggestion
[policy01]: /zh/newsletters/2023/05/17/#等待确认-1-我们为什么需要一个交易池
```

**Review by bitschmidty — CHANGES_REQUESTED** [review]

**Review by bitschmidty — APPROVED** [review]
ACK, thank you @hulatown !

**Comment by bitschmidty** [comment]
@hulatown there were conflicts with this branch so I rebased and squashed but didn’t have permission to push the changes to this branch so I opened a separate branch and PR. Can you take a look? https://github.com/bitcoinops/bitcoinops.github.io/pull/1149

**Comment by hulatown** [comment]
> 

Thanks Mike! It looks good to me

**Comment by bitschmidty** [comment]
Closed in favor of rebased https://github.com/bitcoinops/bitcoinops.github.io/pull/1149

**Pull request closed without merging**